var/spack/repos/builtin/packages/sina/package.py

```diff
@@ -0,0 +1,101 @@
+import os
+
+from spack import *
+
+
+class Sina(CachedCMakePackage):
+    """Sina C++ Library"""
+
+    homepage = 'https://github.com/LLNL/Sina'
+    url = 'https://github.com/LLNL/Sina/releases/download/v1.10.0/sina-cpp-1.10.0.tar.gz'
+
+    version('develop', git='git@github.com:LLNL/Sina.git',
```

Comment:
don't use the `git` protocol URL here; it won't work unless you have ssh set up.  Use the https URL.

Reply:
Turns out we don't even have a develop branch on github :). Most of our work is done internally and then pushed when a release is ready. I will just remove this.